database/sql/V057__add_fee_policy.sql

```diff
@@ -0,0 +1,22 @@
+-- Fee polices represents input parameters for the fee calculation, where fee is taken to compensate the protocol for it's services.
+-- There are different fee policies:
+-- 1. Based on price improvement - if limit order is filled at the price higher than max(limit price, best_quote), then fee is taken as a cut from the price difference. For this type of fee policy, all parameters exist and the fee is minimum of them.
+-- 2. Based on the volume - fee is taken as part of the order volume. For this type of fee policy, only `volume_percentage_factor` exist.
+-- 3. Based on the absolute fee - fee is taken as an absolute value. For this type of fee policy, only `absolute_fee` exist.
+CREATE TABLE fee_policies (
+  auction_id bigint NOT NULL
+  order_uid bytea NOT NULL
+  -- The order in which the fee policies are inserted and applied.
+  insertion_order SERIAL NOT NULL
```

Comment:
The name `insertion_order` seems off. Couldn't the items be inserted in any order? The only thing that matters is the order in which the policies get applied. So maybe `application_order` or just `index`? 🤷‍♂️

Reply:
index is reserved term. switched to `application_order`